pom.xml

```diff
@@ -441,7 +441,7 @@
       <plugin>
         <groupId>org.codehaus.gmaven</groupId>
         <artifactId>gmaven-plugin</artifactId>
-        <version>1.5</version>
+        <version>1.5-jenkins-3</version>
```

Comment:
Uses a more canonical location under `target/` for generated sources, which is helpful for NetBeans at least.